enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -378,14 +387,22 @@ class RemoteKernelManager(EnterpriseGatewayConfigMixin, AsyncIOLoopKernelManager
     returned - upon which methods of poll(), wait(), send_signal(), and kill() can be called.
     """
 
+    event_callbacks = Dict()
+
+    def _event_callbacks_default(self):
+        return dict(
+            kernel_refresh=[], kernel_refresh_failure=[]
+        )  # define new default when adding new event.
+
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.process_proxy = None
         self.response_address = None
         self.public_key = None
         self.sigint_value = None
         self.kernel_id = None
-        self.user_overrides = {}
+        self.user_overrides = {}  # this is populated via create kernel request.
+        self.user_update_overrides = {}  # this is populated via update kernel request.
```

Comment:
Was this feature originally named 'update' or something?  I think `configure_kernel_overrides` or `configure_magic_overrides` is more indicative of where these are coming from.